versions.properties

```diff
@@ -1,87 +1,37 @@
-##                          # available=1.3.0-alpha14
-##                          # available=1.3.0-alpha15
-##                          # available=1.3.0-alpha16
-##                          # available=1.3.0-beta0
-##                          # available=1.3.0
-##                          # available=1.3.1
-##                          # available=1.3.2
-##                          # available=1.3.3
-##                          # available=1.3.4
-##                          # available=1.4.0
-##                          # available=1.4.1
-##                          # available=1.4.2
-##                          # available=1.4.3
-##                          # available=1.4.4
-
-version.ch.qos.logback..logback-core=1.2.11
-##                       # available=1.3.0-alpha0
-##                       # available=1.3.0-alpha1
-##                       # available=1.3.0-alpha2
-##                       # available=1.3.0-alpha3
-##                       # available=1.3.0-alpha4
-##                       # available=1.3.0-alpha5
-##                       # available=1.3.0-alpha6
-##                       # available=1.3.0-alpha7
-##                       # available=1.3.0-alpha8
-##                       # available=1.3.0-alpha9
-##                       # available=1.3.0-alpha10
-##                       # available=1.3.0-alpha11
-##                       # available=1.3.0-alpha12
-##                       # available=1.3.0-alpha13
-##                       # available=1.3.0-alpha14
-##                       # available=1.3.0-alpha15
-##                       # available=1.3.0-alpha16
-##                       # available=1.3.0-beta0
-##                       # available=1.3.0
-##                       # available=1.3.1
-##                       # available=1.3.2
-##                       # available=1.3.3
-##                       # available=1.3.4
-##                       # available=1.4.0
-##                       # available=1.4.1
-##                       # available=1.4.2
-##                       # available=1.4.3
-##                       # available=1.4.4
+plugin.com.github.vlsi.stage-vote-release=1.90
+
+version.assertj.core=3.23.1
+
+version.ch.qos.logback..logback-classic=1.4.11
+
+version.ch.qos.logback..logback-core=1.4.11
 
 version.com.beust..jcommander=1.82
 
-version.com.github.spotbugs..spotbugs=4.7.1
-##                        # available=4.7.2
-##                        # available=4.7.3
+version.com.github.spotbugs..spotbugs=4.8.1
 
 version.com.google.inject..guice-bom=5.1.0
+##                       # available=6.0.0-rc1
+##                       # available=6.0.0-rc2
+##                       # available=6.0.0
+##                       # available=7.0.0-rc1
+##                       # available=7.0.0
 
-version.commons-io..commons-io=2.11.0
+version.commons-io..commons-io=2.15.0
 
 version.in.jlibs..jlibs-core=3.0.1
 
 version.javax..javaee-api=8.0.1
+##            # available=8.0.1-b5
```

Comment:
This cannot be removed since it is being used here https://github.com/testng-team/testng/blob/master/testng-test-osgi/testng-test-osgi-build.gradle.kts#L21

Reply:
I'm talking about the commented version, not the dependency.

Reply:
@juherr - They keep getting added back everytime `refreshVersions` gets executed. 

The whole point of basically doing this exercise was because I am noticing that the publish snapshot is failing because there are uncommitted changes https://github.com/testng-team/testng/actions/runs/6941674340/job/18882998284#step:4:93

I am guessing that somehow `publish` is invoking `refreshVersions`

So can we please get this change merged so that I can figure out if that solves the problem. If not I would have to add `-PallowUncommittedChanges` as suggested in the error message.

Reply:
Ok for this one. I supposed -b was for beta version that is before the release.

Reply:
Even for the other instance where in you had mentioned about removing them, i see the same behaviour. It looks like `refreshVersions` adds those comments back about new versions, the moment it detects them.